backend/http/user.go

```diff
@@ -109,7 +109,16 @@ func (s *Server) handleUserCreate(w http.ResponseWriter, r *http.Request) {
 		s.error(w, r, err)
 		return
 	}
-	s.Logger.Infof("created user %q (id = %q)", user.Name, user.ID)
+	s.Logger.Infof("created user %q (id = %d)", user.Name, user.ID)
```

Comment:
Changed the verb from %q to %d as well because %q on a number outputs the hex code instead of the decimal value